lib/ts/recipe/accountlinking/types.ts

```diff
@@ -85,12 +85,15 @@ export type RecipeInterface = {
         recipeId: string;
         timeJoined: number;
         userContext: any;
-    }) => Promise<{
-       status: "OK",
```

Comment:
for the onAccountUnlinked function, we should add an argument like `wasRecipeUserDeleted: boolean`